pandas/core/indexes/base.py

```diff
@@ -2788,26 +2788,45 @@ def set_value(self, arr, key, value):
 
     def _get_level_values(self, level):
         """
+        Return an Index of values for requested level, specially useful for
+        MultiIndex.
+
         Return an Index of values for requested level, equal to the length
-        of the index.
+        of the index. This is mainly useful for MultiIndex.
 
         Parameters
         ----------
         level : int or str
-            ``level`` is either the integer position of the level in the
-            MultiIndex, or the name of the level.
+            It is either the integer position or the name of the level.
+            If given as an integer, it must be between 0 and the
```

Comment:
Ah this isn't quite right. Negative indices are accepted as well. Probably OK to just remove the second sentence, as to most Python programmers, "integer position" is clear enough.